lib/magnet/parser.rb

```diff
@@ -2,7 +2,7 @@ module Magnet
   class Parser
     TRACKS = {
       1 => /\A%(?<format>[A-Z])(?<pan>[0-9 ]{1,19})\^(?<name>[^^]*)\^\s?(?<expiration>\d{4}|\^)(?<service_code>\d{3}|\^)(?<discretionary_data>[^\?]*)\?\Z/,
```

Comment:
If it won't be used, I wouldn't bother. We can add it when we need the feature and have a real test case.